dmoj/executors/filesystem_policies.py

```diff
@@ -0,0 +1,114 @@
+import os
```

Comment:
This file should probably live inside `dmoj.cptbox` rather than `dmoj.executors`.